docs/av-stack-sync.1.md

```diff
@@ -0,0 +1,61 @@
+# av-stack-sync 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-sync - Synchronize stacked branches
```

Comment:
```suggestion
av-stack-sync - Synchronize stacked branches from their parent branches
```

Reply:
Discussed offline, but we do not use "synchronize from".